tests/test_cli.py

```diff
@@ -27,3 +31,17 @@ def test_dockerize_command(monkeypatch, capsys):
     os.remove(dummy_server_file)
     assert "Dockerfile created successfully" in captured.out, "CLI did not create Dockerfile"
     assert os.path.exists("Dockerfile"), "CLI did not create Dockerfile"
+
+
+@patch("importlib.util.find_spec")
+@patch("subprocess.check_call")
+def test_ensure_lightning_installed(mock_check_call, mock_find_spec):
+    mock_find_spec.return_value = False
+    _ensure_lightning_installed()
+    mock_check_call.assert_called_once_with([sys.executable, "-m", "pip", "install", "-U", "lightning-sdk"])
+
+
+def test_lightning_serve_help():
```

Comment:
can we also have a test that all other SDK commands work?

Reply:
we just install the SDK without making any change